lib/ui/platform_dispatcher.dart

```diff
@@ -149,6 +149,26 @@ class PlatformDispatcher {
     _onPlatformConfigurationChangedZone = Zone.current;
   }
 
+  /// The current list of displays.
+  ///
+  /// If any of their configurations change, [onMetricsChanged] will be called.
+  ///
+  /// To get the display for a [FlutterView], use [FlutterView.display].
+  ///
+  /// Platforms may limit what information is available to the application with
+  /// regard to secondary displays and/or displays that do not have an active
+  /// application window.
+  ///
+  /// Presently, on Android and Web this collection will only contain the
+  /// display that the current window is on. On iOS, it will only contains the
+  /// main display on the phone or tablet. On Desktop, it will contain only
+  /// a main display with a valid refresh rate but invalid size and device
+  /// pixel ratio values.
```

Comment:
Is it possible to make these values missing or null instead of invalid (which I assume means incorrect).

Reply:
I don't want to make them null, since it will be harder to migrate them away from nullable later if people start treating them as nullable (e.g. they add `!` all over and the compiler refuses to compiile things when we switch it).

The plan is to implement them for desktop sooner than later.